includes/class-wc-gateway-swedbank-pay-checkout.php

```diff
@@ -829,7 +833,58 @@ public function capture_payment( $order, $amount = false ) {
 			return;
 		}
 
-		parent::capture_payment( $order, $amount );
+		//parent::capture_payment( $order, $amount );
+
+		$paymentorder_id = get_post_meta( $order->get_id(), '_payex_paymentorder_id', true );
+		$result = $this->request( 'GET', $paymentorder_id );
+		$capture_href = self::get_operation( $result['operations'], 'create-paymentorder-capture' );
+		if ( empty( $capture_href ) ) {
+			throw new Exception( __( 'Capture unavailable', WC_Swedbank_Pay_Checkout::TEXT_DOMAIN ) );
```

Comment:
A missing `create-paymentorder-capture` is not necessarily an error, so please don't throw an exception here. For one-phase payment instruments like Swish, there won't be a `create-paymentorder-capture` operation, but you should find a `paid-paymentorder` or a `failed-paymentorder` operation that will tell you whether the payment was successful (`paid`) or not (`failed`).

Reply:
@asbjornu We shouldn't show page from ```paid-paymentorder``` because this method uses for WooCommerce admin backend. This methods uses for all payment methods placed with the checkout module, not Swish only

Reply:
`paid-paymentorder` is an API operation, not something to show anyone. Is the code we're looking at here only used in the WooCommerce admin backend? Would it be possible to disable the "Capture" button if the `create-paymentorder-capture` operation isn't found?